src/cli.ts

```diff
@@ -96,6 +102,7 @@ const cwd = process.cwd();
  */
 const sourcePath = normalizePath(argv.flags.src, true);
 const distPath = normalizePath(argv.flags.dist, true);
+const tsconfig = getTsconfig(undefined, argv.flags.tsconfig);
```

Comment:
`parseTsconfig` should be used here. `getTsconfig` searches for a tsconfig.json file, but `parseTsconfig` parses the tsconfig.json at a given path.

Example:
https://github.com/esbuild-kit/cjs-loader/blob/94a69f9e6d41225cb8a132938a3b9b077c15f966/src/index.ts#L23